package-lock 2.json

```diff
@@ -0,0 +1,14571 @@
+{
```

Comment:
I haven't seen a `package-lock2.json` file before - can you help me understand why there are two of these files in the project?

Reply:
Hey Mariah - great catch! This was purely just and accident 😅whoops. I removed and pushed up the changes.